pkg/minikube/registry/drvs/kvm2/kvm2.go

```diff
@@ -86,7 +90,32 @@ func configure(mc config.MachineConfig) interface{} {
 	}
 }
 
+// check whether current user is in either libvirt or libvirtd group
+func isUserInVirtGroup() (bool, string) {
+	vid, _ := group.LookupGid("libvirt")
```

Comment:
Let's use go stdlib only here. No need to add an extra dependency.

Reply:
Code refactored to use os.user package